muselog/datadog.py

```diff
@@ -126,20 +126,25 @@ def json_record(self, message, record):
 
         record_dict['message'] = message
 
-        if "context" in record_dict:
-            context_value = record_dict.get("context")
-            array = context_value.replace(" ", "").split(",")
-            for item in array:
-                key, val = item.split("=")
-
-                # del key from record before replacing with modified version
-                del record_dict[key]
-
-                key = f"ctx.{key}"
-                context_obj[key] = int(val) if val.isdigit() else val
-                record_dict.update(context_obj)
-
-            del record_dict['context']
+        try:
```

Comment:
This may work as a quickfix, but ultimately we should not be seeing these errors in the first place. In the interest of time, I'll explain the issue. The issue is that certain companies have a comma in their name, such as those ending in `, INC.`. There are (at least) two approaches to handle this. I'll give you some time to figure those approaches out (after you're feeling better). 

For now, this will work, but expect it to drop logs.